api/api-rules/violation_exceptions.list

```diff
@@ -219,6 +219,7 @@ API rule violation: list_type_missing,k8s.io/api/storage/v1,CSINodeSpec,Drivers
 API rule violation: list_type_missing,k8s.io/api/storage/v1,CSIStorageCapacityList,Items
 API rule violation: list_type_missing,k8s.io/api/storage/v1,StorageClass,MountOptions
 API rule violation: list_type_missing,k8s.io/api/storage/v1alpha1,CSIStorageCapacityList,Items
+API rule violation: list_type_missing,k8s.io/api/storage/v1alpha1,VolumeAttributesClassList,Items
```

Comment:
@liggitt @msau42 Is this expected? I see other Items types listed here.. but the violation is for the list type being missing :neutral_face:

Reply:
@jpbetz It doesn't need add new item after I drop the `//+ listXXX` comment.